### DISCUSSION TITLE
uxm_setup.setup_phase_delay does not use the modify_attens

### DISCUSSION THREAD

**Discussion by yaqiongl** [simonsobs/sodetlib#350] [Issues and Debugging] [answered]
This is a really small issue, it seems uxm_setup.setup_phase_delay does not use the args modify_attens at all, and would change the uc and dc attens anyway with modify_attens=False
It wouldn't cause any trouble if just doing uxm_setup.uxm_setup, it just causes a bit confusion if someone runs this function individually.

**Comment by jlashner** [comment, marked as answer]
Thanks for posting this Yaqiong, we had a bit of a discussion / conflict of what "modify_attens" should mean in this PR: https://github.com/simonsobs/sodetlib/issues/262.

We ended up deciding that since `uxm_setup` only should really run ~once at the beginning of each cooldown, it doesn't really need to support running with pre-configured attens, and `modify_attens=False` means "Don't modify the cfg atten values" (and as long as the docstring accurately describes what's happening it's fine).

Good catch about an unused `modify_attens` argument in setup phase delay possibly causing confusion. I'll remove that and submit a PR.

**Comment by jlashner** [comment]
Just submitted https://github.com/simonsobs/sodetlib/pull/352 which removes the keyword and updates docs to better describe the current behavior.
